docs/load-test-how-to-use.md

```diff
@@ -0,0 +1,119 @@
+ - Make the necessary changes in `loadtest/`
+
+#### Testing changes locally
+
+ - Go through [local_loadtest.md](https://github.com/mattermost/mattermost-load-test-ng/blob/master/docs/local_loadtest.md)
+ - Some additional information on the above document
+    - In step where configs are copied from samples, `simplecontroller.json` doesn't have to be generated since by default the `config.json` uses a `simulative` controller. <!-- docfix -->
+    - updates to `config.json`
+        - Make sure to change `ConnectionConfiguration` section in config.json according to the local deployment of mattermost-server.
+        - `InstanceConfiguration` refers to [doc](https://github.com/mattermost/mattermost-load-test-ng/blob/master/docs/loadtest_config.md#instanceconfiguration), this is used by `init` command later to initially populate mattermost database.
+    - increase frequency of the new action, so its easier to debug while running locally. [sample](https://github.com/mattermost/mattermost-load-test-ng/blob/8faa4dfb485dace3bd65908c0d3d98979b7dfd17/loadtest/control/simulcontroller/controller.go#L227)
+    - Expect some failures in [running-a-basic-loadtest](https://github.com/mattermost/mattermost-load-test-ng/blob/master/docs/local_loadtest.md#running-a-basic-load-test), if one sees > 10-15 error logs, there's a [troubleshooting guide](https://github.com/mattermost/mattermost-load-test-ng/blob/master/docs/faq.md#troubleshooting).
+    - check `ltagent.log` in `mattermost-load-test-ng` directory, and server logs for details on errors, if any.
+    - Further sections of the document i.e [using load-test agent API server](https://github.com/mattermost/mattermost-load-test-ng/blob/master/docs/local_loadtest.md#running-a-load-test-through-the-load-test-agent-api-server) and [using load-test coordinator](https://github.com/mattermost/mattermost-load-test-ng/blob/master/docs/local_loadtest.md#running-a-load-test-through-the-coordinator) are highly recommended to go through, since the Terraform deployment uses the latter method to execute the load-tests and it'd be easier to debug the setup if the developer understands these underlying principles.
+
+
+#### Testing changes in terraform
+
+Even without the framework, a general load-test workflow in the cloud will be similar to the following.
+
+ - Create a database (to be used by mattermost servers)
+ - Create deployments of mattermost servers - let's call them app servers for convenience.
+ - Create 'agent' machines to ping app-servers using [controllers](https://github.com/mattermost/mattermost-load-test-ng/tree/61c44f35224b76d3098199b0cd2b67db2222b549/loadtest/control)
+ - Populate database, and start app, agents (i.e loadtest)
+ - Collect metrics from app, and agent deployments to either,
+    - manage an unbounded load test
+    - analyse api performance after the load-test completes.
```

Comment:
I think all this is good information but maybe makes more sense at the bottom in another section called something like "Test changes in the cloud without terraform". What do you think?